src/init/kafka_exporter.rs

```diff
@@ -0,0 +1,347 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::exporters::kafka::config::{
+    AcknowledgementMode, KafkaExporterConfig, PartitionerType, SerializationFormat,
+};
+use crate::init::parse::parse_key_val;
+use clap::{Args, ValueEnum};
+use serde::Deserialize;
+
+#[derive(Debug, Args, Clone, Deserialize)]
+pub struct KafkaExporterArgs {
+    /// Kafka broker addresses (comma-separated)
+    #[arg(
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_BROKERS",
+        default_value = "localhost:9092"
+    )]
+    pub kafka_exporter_brokers: String,
+
+    /// Topic name for traces
+    #[arg(
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_TRACES_TOPIC",
+        default_value = "otlp_traces"
+    )]
+    pub kafka_exporter_traces_topic: String,
+
+    /// Topic name for metrics
+    #[arg(
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_METRICS_TOPIC",
+        default_value = "otlp_metrics"
+    )]
+    pub kafka_exporter_metrics_topic: String,
+
+    /// Topic name for logs
+    #[arg(
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_LOGS_TOPIC",
+        default_value = "otlp_logs"
+    )]
+    pub kafka_exporter_logs_topic: String,
+
+    /// Serialization format
+    #[arg(
+        value_enum,
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_FORMAT",
+        default_value = "json"
+    )]
+    pub kafka_exporter_format: KafkaSerializationFormat,
+
+    /// Compression type (gzip, snappy, lz4, zstd, none)
+    #[arg(long, env = "ROTEL_KAFKA_EXPORTER_COMPRESSION")]
+    pub kafka_exporter_compression: Option<String>,
+
+    /// Acknowledgement mode (none, one, all)
+    #[arg(
+        value_enum,
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_ACKS",
+        default_value = "one"
+    )]
+    pub kafka_exporter_acks: KafkaAcknowledgementMode,
+
+    /// Client ID for the Kafka producer
+    #[arg(long, env = "ROTEL_KAFKA_EXPORTER_CLIENT_ID", default_value = "rotel")]
+    pub kafka_exporter_client_id: String,
+
+    /// Maximum message size in bytes
+    #[arg(
+        long,
+        env = "ROTEL_KAFKA_EXPORTER_MAX_MESSAGE_BYTES",
+        default_value = "1000000"
+    )]
+    pub kafka_exporter_max_message_bytes: usize,
+
+    /// Linger time in milliseconds
+    #[arg(long, env = "ROTEL_KAFKA_EXPORTER_LINGER_MS", default_value = "5")]
+    pub kafka_exporter_linger_ms: u32,
```

Comment:
Are these duration fields named to match the OTEL collector then? Alternatively we could match the other exporter configs that use the duration format like `5ms`, but I can see if this is to maintain parity.